server/src/main/kotlin/org/javacs/kt/KotlinLanguageServer.kt

```diff
@@ -141,6 +143,11 @@ class KotlinLanguageServer(
         InitializeResult(serverCapabilities, serverInfo)
     }
 
+    private fun clientHasWorkspaceSymbolResolveSupport(clientCapabilities: ClientCapabilities) =
+        clientCapabilities?.workspace?.symbol?.resolveSupport?.properties?.let { properties ->
+            if (properties.size > 0) SymbolResolveSupport(true, properties) else null
+        } ?: SymbolResolveSupport(false, emptyList())
```

Comment:
Nit: If the function does not return a boolean, its name should not be `clientHas...`. Though if we pass the client capabilities to `KotlinWorkspaceService`, I would either

- remove this function (since we only use it in one place anyway)
- move `SymbolResolveSupport` to a separate file (e.g. in `org.javacs.kt.symbols`) and make this an extension property on `ClientCapabilities`, i.e. something along the lines of
  ```kotlin
  val ClientCapabilities.symbolResolveSupport
    get() = ...
  ```

Reply:
I went for the SymbolResolveSupport. I think technically more properties can me masked in the symbol calls in the future although the only example of this is the location property, based on the types available for return.